File: src/main/java/com/pCarpet/model/Stop.java
package com.pCarpet.model;


import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotNull;

import lombok.*;

//@Data

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor

@Entity
public class Stop {
	
	@Id	
	@Column
	@GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id_stop;
	
	@Column
    private String start;
	
	@Column
    private String finish;
	
	@Column
	@NotNull
    private Boolean expired;	
	
	@ManyToOne
    @JoinColumn(name="id_car")
	private Car car;
	
	@ManyToOne
    @JoinColumn(name="id_slot")
	private Slot slot;
	
}   	
	
  

File: src/app/core/models/Stop.ts


export class Stop {
    id_stop: number;
    address: string;
    start: Date;
    finish: Date;
    name: string;
    price: number;
    //servono?
//    id_car: number;
//    id_slot: number;

    constructor(id: number, address: string, start: Date, finish: Date, name:string, price: number){
            this.id_stop = id;
            this.address = address;
            this.start = start;
            this.finish = finish;
            this.name = name;
            this.price = price;
//            this.id_car = id_car;
//            this.id_slot = id_slot;

    }
}